samples/snippets/cpp/VS_Snippets_Remoting/NCLWebClientAsync/CPP/asyncmethods.cpp

```diff
@@ -420,19 +378,15 @@ void DownLoadFileInBackground2( String^ address )
 {
    WebClient^ client = gcnew WebClient;
    Uri ^uri = gcnew Uri(address);
-   
-   // Specify that the DownloadFileCallback method gets called
-   // when the download completes.
+
+   // Call DownloadFileCallback2 when the download completes.
    client->DownloadFileCompleted += gcnew AsyncCompletedEventHandler( DownloadFileCallback2 );
-   
-   // Specify a progress notification handler.
-   client->DownloadProgressChanged += gcnew DownloadProgressChangedEventHandler( DownloadProgressCallback );
+
+   // Specify a progress notification handler here...

```

Comment:
I removed the line that adds the DownloadProgressChanged handler since it doesn't apply to the event where this snippet is used (DownloadFileCompleted), and that way we don't have to show the implementation of DownloadProgressCallback here either.